src/ui/public/directives/paginate.js

```diff
@@ -40,6 +40,12 @@ uiModules.get('kibana')
     },
     controllerAs: 'paginate',
     controller: function ($scope, $document) {
+      // Unfortunately this controller relies on inherited scope. Lets not fail if onPageChanged happens to not be
+      // implemented.
+      if (!$scope.onPageChanged) {
```

Comment:
The likelihood of this actually happening is rather slim, and since we'll hopefully be migrating to a new pager soon, it's prolly not that big of a deal.